nix/modules/lanzaboote.nix

```diff
@@ -123,6 +125,15 @@ in
     boot.loader.external = {
       enable = true;
       installHook = pkgs.writeShellScript "bootinstall" ''
+        ${optionalString cfg.generateKeysIfNotExist ''
+          if [ ! -f "${cfg.privateKeyFile}" ]; then
+            mkdir -p ${cfg.pkiBundle}
+            ${sbctlWithPki}/bin/sbctl create-keys \
```

Comment:
```suggestion
            ${getExe sbctlWithPki} create-keys \
```

Reply:
After looking up lib.getExe and verifying that sbctl does provide `mainProgram` I do believe that your change makes sense.

I am unsure if I should forego the prescient coding style in this file. May need an extra PR to fix.
Opinions?